packages/neon_framework/lib/src/models/account.dart

```diff
@@ -32,14 +33,18 @@ class Account implements Credentials, Findable {
     required this.username,
     this.password,
     this.userAgent,
-  }) : client = NextcloudClient(
-          serverURL,
-          loginName: username,
-          password: password,
-          appPassword: password,
-          userAgentOverride: userAgent,
-          cookieJar: CookieJar(),
-        );
+  }) : id = _buildId(username, serverURL) {
+    client = NextcloudClient(
+      serverURL,
+      loginName: username,
+      password: password,
+      appPassword: password,
+      userAgentOverride: userAgent,
+      cookieJar: PersistCookieJar(
+        storage: FileStorage('.cookies/$id'),
```

Comment:
Shouldn't we point to the application cache dir? Or is that automatically prepended?

Reply:
I just used the same approach the `PersistCookieJar` was defaulting to.
If you prefer we can keep the in memory version for now and just move to sqlite directly. I should have the time to do this in the coming days.

Reply:
Yeah that would be better